src/Magnum/Audio/Buffer.h

```diff
@@ -83,7 +89,27 @@ class Buffer {
              *
              * @requires_al_extension Extension @al_extension{EXT,double}
              */
-            StereoDouble = AL_FORMAT_STEREO_DOUBLE_EXT
+            StereoDouble = AL_FORMAT_STEREO_DOUBLE_EXT,
+
+            Quad8 = AL_FORMAT_QUAD8,
```

Comment:
@Alicemargatroid I think we mean different headers: you did add them to `OpenAL/extensions.h`, yes, and that is correct, but you will also need to add them to [Magnum/Audio/Extension.h](https://github.com/mosra/magnum/blob/master/src/Magnum/Audio/Extensions.h#L81), which wraps the extensions in namespaces. This helps with code assists and type safety for example.